### ISSUE TITLE
ember-concurrency should be a dependency

### ISSUE THREAD

**Issue by 0xadada** [oskarrough/ember-youtube#52]
This package doesn't list `ember-concurrency` as a dependency, but it should, since the youtube component it exposes directly consumes it.

from the addon docs:

> Addons can also use other addons as dependencies. For example, this partial package.json is from the addon ember-power-select. It uses the addons ember-concurrency, ember-text-measurer, and ember-truth-helpers to support its functionality.

> In this case, Ember CLI will incorporate these addons along with the other dependencies when ember-power-select is used in your applications.

So, if the consuming application uses the youtube component from `ember-youtube`, and that component imports `ember-concurrency`, it is a direct dependency and should be included as a `dependency` not a `devDependency`.

This is a problem in the case where the consuming application has tests or another addon with a dummy app consumes `ember-youtbe`. 

This can be seen with the following type of error from my app:
```bash
not ok 202 Chrome 74.0 - [undefined ms] - Global error: Uncaught InvalidStateError: Failed to read the 'responseText' property from 'XMLHttpRequest': The value is only accessible if the object's 'responseType' is '' or 'text' (was 'json'). at http://localhost:7357/31425136537553/tests/index.html?hidepassed, line 96
 After execution of test: ember-qunit: Ember.onerror validation: Ember.onerror is functioning properly
    ---
        Log: |
            { type: 'warn',
              text:
               '\'Error occurred while evaluating `mir/components/ember-youtube`: Could not find module `ember-concurrency` imported from `ember-youtube/components/ember-youtube`\\nError: Could not find module `ember-concurrency` imported from `ember-youtube/components/ember-youtube`\\n    at missingModule (http://localhost:7357/assets/vendor.js:252:11)\\n    at findModule (http://localhost:7357/assets/vendor.js:263:7)\\n    at Module.findDeps (http://localhost:7357/assets/vendor.js:173:24)\\n    at findModule (http://localhost:7357/assets/vendor.js:267:11)\\n    at Module.findDeps (http://localhost:7357/assets/vendor.js:173:24)\\n    at findModule (http://localhost:7357/assets/vendor.js:267:11)\\n    at requireModule (http://localhost:7357/assets/vendor.js:29:15)\\n    at http://localhost:7357/31425136537553/tests/index.html?hidepassed:49:11\\n    at Array.forEach (<anonymous>)\\n    at sendCoverage (http://localhost:7357/31425136537553/tests/index.html?hidepassed:44:33)\'\n' }
```

The fix is to move these external dependencies to `devDependencies`.

I've created a new ember app that adds this addon, and reproduces the bug:
https://github.com/0xadada/reproduction-ember-youtube-example
```
Uncaught Error: Could not find module `ember-concurrency` imported from `ember-youtube/components/ember-youtube`
    at missingModule (loader.js:247)
    at findModule (loader.js:258)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
    at requireModule (loader.js:24)
    at Class._extractDefaultExport (index.js:422)
    at Class.resolveOther (index.js:103)
    at Class.resolve (index.js:163)
```

**Comment by 0xadada** [comment]
@oskarrough could you publish a new version to NPM?

**Comment by oskarrough** [comment]
@0xadada yes, published as 0.9.4. Thanks again!
